pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -2642,3 +2642,9 @@
   tutorial_khanweb_string_detail_programming_languages: "HTML & CSS"
   tutorial_khandata_string_detail_programming_languages: "SQL"
   tutorial_robomeshiq_string_detail_programming_languages: "Blocks, Python"
+
+  school_not_found_heading: "We couldn't find your school. Please enter more information."
+  school_name: "School name"
+  school_city: "School city"
+  school_state: "School state"
+  school_zip: "Postal code (if known)"
```

Comment:
I think these should all be capitalized. Like `School Name` and `Postal Code (if known)`. Does Postal Code need the (if known), is it not just optional?

Reply:
There was note on the spec that (if known) should be included, but I don't know where we actually landed on including it or not. I'll confirm with Miranda.